### PR TITLE
Merge release/8.2 to trunk after default branch rename

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR is not strictly necessary and doesn't have any code changes. However, if one tries to merge current `release/8.2` to `trunk` it'd show as if there are changes in the release branch: https://github.com/woocommerce/woocommerce-android/compare/release/8.2?expand=1

This is because we applied release-toolkit update to both `release/8.2` and `trunk`, release branch having a smaller set of changes. Without a merge commit from `release/8.2` to `trunk`, GitHub is not able to tell `trunk` has already have the same changes applied.

In short, this PR is just my attempt at removing any future confusion by the release manager.

### CODE REVIEW

**Review by loremattei — APPROVED** [review]
An easy one :-)
